com.microsoft.mrtk.extendedassets/Animations/Earth/EarthActiveHover.anim

```diff
@@ -0,0 +1,179 @@
+%YAML 1.1
+%TAG !u! tag:unity3d.com,2011:
+--- !u!74 &7400000
```

Comment:
Is there a reason why these are "earth-model"-specific? And, if they are earth-specific, they should probably be checked in where the Earth prefab is (which is the sample project, not EA, right?)

Reply:
Great point, since it works with other objects as well. Renamed it to ObjectActiveHover/PassiveHover/Select. Earth prefab is in EA.